source/ui/beamline/AMExtendedControlEditor.cpp

```diff
@@ -456,6 +462,20 @@ QSize AMExtendedControlEditor::sizeHint() const{
 	return newHint;
 }
 
+void AMExtendedControlEditor::onControlMinimumValueChanged()
+{
+	if (control_) {
```

Comment:
That's the problem this pull request attempts to solve, it's the present behavior. With the changes to AMPVControl, controls should emit their min and max values when they connect. But in case the editor is created after they are connected, the editor should be able to ask a control for its min/max values.